### PR TITLE
release 0.7.1

### PR DESCRIPTION
**Pull request by xflr6**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  fn: python-graphviz-{{ version }}.tar.gz
-  url: https://github.com/xflr6/graphviz/archive/{{ version }}.tar.gz
+  fn: graphviz-{{ version }}.zip
+  url: https://files.pythonhosted.org/packages/source/g/graphviz/graphviz-{{ version }}.zip
   sha256: {{ sha256 }}
   patches:
     # needed due to the Windows batch script wrappers in conda-forge Graphviz
     # see https://github.com/conda-forge/graphviz-feedstock/blob/bbfe3c7be2a448dd11db165a18c3edfd5ee6a95d/recipe/bld.bat#L27-L32
-    - windows-bat.patch
+    - windows-bat.patch  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml:16]: Could you please revert this change?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,7 +23,6 @@ requirements:
   build:
     - python
     - setuptools
-
```
Comment (jakirkham) [inline recipe/meta.yaml]: Also please revert this one.

**Comment by jakirkham** [comment]
Two minor comments. Otherwise LGTM.

File: recipe/meta.yaml (hunk 1 above)
Comment (xflr6) [reply recipe/meta.yaml:16]: `patch` generally does not work on files with CRLF line endings (see the source archive) unless you use its `--binary` option, which cannot be configured here (see conda/conda-build#1973). Differing from the linux and darwin version, the Windows-version actually converts the line endings (may be a compile-time option or the default for `patch` as delivered  by git-for-windows).

IOW, removing this makes the linux and os x builds fail (also, this is the smallest possible change so one might prefer it for independent reasons: only Windows Anaconda has the `.bat`-wrappers).

Comment (jakirkham) [reply recipe/meta.yaml:16]: Ok, that's fine. Didn't know we had issues with this as we already had the patch applied on all platforms before.

**Comment by jakirkham** [comment]
Thanks @xflr6.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+{% set number = "0" %}
 
 package:
   name: python-graphviz
   version: {{ version }}
 
 source:
-  fn: python-graphviz-{{ version }}.tar.gz
-  url: https://github.com/xflr6/graphviz/archive/{{ version }}.tar.gz
+  fn: graphviz-{{ version }}.zip
```
Comment (jakirkham) [inline recipe/meta.yaml:10]: I think we should keep the `python-` here as well to distinguish this source from that of the `graphviz` CLI/library.

Comment (jakirkham) [reply recipe/meta.yaml:10]: Addressing in PR ( https://github.com/conda-forge/python-graphviz-feedstock/pull/10 ).

Comment (xflr6) [reply recipe/meta.yaml:10]: -1: this is the 'offifical' name of the source distribution file (would you rename the file on download when building by hand?).

File: recipe/meta.yaml (hunk 1 above)
Comment (xflr6) [reply recipe/meta.yaml:16]: Yes, that was with the archive from GitHub, which has LF line endings due to conversion from git.. :)

File: recipe/meta.yaml (hunk 3 above)
Comment (jakirkham) [reply recipe/meta.yaml:10]: The point is one might be building `graphviz` and `python-graphviz` locally. In this case, the name here merely helps disambiguate them the two tarball in `conda-build`'s source cache nothing more. The filename has no meaning outside of that.
